x-pack/legacy/plugins/apm/public/index.tsx

```diff
@@ -57,5 +31,5 @@ const checkForRoot = () => {
   });
 };
 checkForRoot().then(() => {
-  plugin().start(core);
+  plugin().start(core, plugins);
 });
```

Comment:
is it possible to use `async`/`await` here?

Reply:
You can only `await` inside of an `async` function, so if we wanted to do that here we would need to wrap everything inside an immediately executing async function, which would probably be just as complex as what we're doing already.